tests/integration/tests.py

```diff
@@ -601,6 +602,9 @@ class CspReportTest(TestCase, SnubaTestCase):
     def assertReportCreated(self, input, output):
         resp = self._postCspWithHeader(input)
         assert resp.status_code == 201, resp.content
+        # XXX: there appears to be a race condition between the 201 return and get_events,
+        # leading this test to sometimes fail. .1s seems to be sufficient.
```

Comment:
could you please just add a comment explaining we cannot change the timestamp of store_event as we do in other SNuba tests because the event is not created directly by this test ?

Reply:
Yeah.